examples/website-demos/social-messaging/src/assets/ArrowLeft.tsx

```diff
@@ -1,5 +1,24 @@
-export const ArrowLeft: React.FC = () => (
-    <svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
-<path d="M16 6.99999V8.99999H3.98955L9.49477 14.5052L8.08057 15.9194L0.161133 7.99999L8.08057 0.0805664L9.49477 1.49478L3.98955 6.99999H16Z" fill="black"/>
-</svg>
+type Props = {
+    isNewChat: boolean;
+    setNewChat: React.Dispatch<React.SetStateAction<boolean>>;
+};
```

Comment:
for all the icons, would prob be easier just to pull these from an AppContext or something like that, you can see how we did that in the event app

Reply:
Is this implementation in the event app pushed up to the website-demos branch?